apps/sync-javascript/src/portabl.js

```diff
@@ -12,34 +12,87 @@ if (!MOCK_USER_ID) {
   localStorage.setItem('MOCK_USER_ID', MOCK_USER_ID);
 }
 async function createMockProviderInputs(mockUserId) {
-  let MOCK_HEADERS_WITH_AUTH = { Authorization: `Basic ${window.btoa(mockUserId)}` };
+  const MOCK_HEADERS_WITH_AUTH = { Authorization: `Basic ${window.btoa(mockUserId)}` };
+
   const { data: claims } = await axios.get(`${API_BASE_URL}${CLAIMS}`, {
     headers: MOCK_HEADERS_WITH_AUTH,
   });
+
   const claimForm = document.querySelector('.claim-form');
 
-  const emailInputEl = claimForm.querySelector("input[data-claim-key='email']");
-  emailInputEl.value = claims.emailAddress;
+  const firstNameInputEl = claimForm.querySelector("input[data-claim-key='firstName']");
```

Comment:
Now that we're adding so many inputs, how do you feel about creating an object to map through them for easier legibility?